### PR TITLE
UIEH-1384 *BREAKING* Bump `react` to `v18`

### PR DESCRIPTION
**Pull request by BogdanDenis**
## Description
*BREAKING* Bump react to v18

## Issues
[UIEH-1384](https://issues.folio.org/browse/UIEH-1384)

### CODE REVIEW

File: src/routes/package-edit-route/package-edit-route.test.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import {
   cleanup,
   act,
```
Comment (Dmytro-Melnyshyn) [inline src/routes/package-edit-route/package-edit-route.test.js:6]: Please fix all unused variables in the ui-eholdings.

File: package.json [hunk 2]
```diff
@@ -25,41 +25,31 @@
...
+    "@folio/stripes": "^9.0.0",
+    "@folio/stripes-cli": "^3.0.0",
+    "@folio/stripes-core": "^10.0.0",
     "@formatjs/cli": "^4.2.20",
-    "@testing-library/jest-dom": "^5.11.1",
-    "@testing-library/react": "^11.0.2",
-    "@testing-library/react-hooks": "^5.1.2",
     "axe-core": "~4.1.4",
-    "babel-jest": "^26.3.0",
-    "babel-polyfill": "^6.26.0",
```
Comment (Dmytro-Melnyshyn) [inline package.json:45]: `babel-polyfill` is used in `test/bigtest/index.js`

File: jest.config.js [hunk 3]
```diff
@@ -1,28 +1,10 @@
...
+  ...config,
+  setupFiles: [
+    ...config.setupFiles,
+    path.join(__dirname, './test/jest/setup-tests.js'),
   ],
-  coverageDirectory: './artifacts/coverage-jest/',
-  coverageReporters: ['lcov'],
-  reporters: ['jest-junit', 'default'],
-  transform: {
-    '^.+\\.(js|jsx)$': path.join(__dirname, './test/jest/jest-transformer.js'),
```
Comment (Dmytro-Melnyshyn) [inline jest.config.js:16]: `jest-transformer.js` should be removed too.

**Comment by Dmytro-Melnyshyn** [comment]
`babel.config.js` has to be removed too.

**Review by Dmytro-Melnyshyn — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (BogdanDenis) [reply package.json:45]: Bigtest tests are not used anymore, I created a ticket to remove everything related to it

**Review by Dmytro-Melnyshyn — APPROVED** [review]

**Pull request merged**